tools/authorPatchCollector.py

```diff
@@ -0,0 +1,109 @@
+import argparse
+import subprocess
+
+
+def patch_collector(dir_path, authors, output):
+    print(f'\n**Inside {dir_path}**\n')
+    end = ''
+    begin = ''
+    patch_count = 1
+    range_count = 1
+
+    author_list = ''
+    for author in authors:
+        author_list += f'--author={author} '
+
+    get_hashes_cmd = (f'git -C {dir_path} log --pretty=format:"%h" '
+                      f'{author_list}')
+    hashes = subprocess.check_output(get_hashes_cmd, shell=True)
+    hashes = hashes.decode(encoding='utf-8')
+    hashes = hashes.split('\n')
+
+    for hash in hashes:
```

Comment:
hash is a reserved word

use commit_hash ?

Reply:
Sure!